src/portalbackend/PortalBackend.DBAccess/Repositories/OfferRepository.cs

```diff
@@ -350,8 +350,7 @@ sorting switch
                 o.UserRoles.Any(),
                 o.Documents.Where(doc => doc.DocumentStatusId != DocumentStatusId.LOCKED )
```

Comment:
this should be changed to 

```suggestion
                o.Documents.Where(doc => doc.DocumentStatusId == DocumentStatusId.PENDING)
```

otherwise we would reaneble already deleted (INACTIVE) documents

Reply:
fixed